src/css/custom.css

```diff
@@ -6,28 +6,28 @@
 
 /* You can override the default Infima variables here. */
 :root {
-  --ifm-color-primary: #5A8CE9;
-  --ifm-color-primary-dark: #5A8CE9;
+  --ifm-color-primary: #5a8ce9;
```

Comment:
it's the same value but in lowercase. For what do we need this change?

Reply:
formatting

Reply:
I don't think it's auto-formatted(see the line 11) and UPPERCASE is more popular for HEX-colors